src/client/components/input/partial-date.js

```diff
@@ -88,6 +121,10 @@ class PartialDate extends React.Component {
 		return validationClass;
 	}
 
+	/**
+	* Renders the date input field in the form.
```

Comment:
Not necessarily in a form. Just "Renders the PartialDate input field" would work here.